enatega-multivendor-web/src/components/Footer/Footer.js

```diff
@@ -98,15 +98,15 @@ function Footer() {
               {t("footerLinkHome")}
             </Typography>
           </RouterLink>
-          <RouterLink to="/privacy" style={{ textDecoration: "none" }}>
+          <RouterLink to="/privacy" onClick={handleButtonClick} style={{ textDecoration: "none" }}>
```

Comment:
Applying button click to privacy and terms links are useless as they already redirect to other pages, So we don't need to scroll to top as new page already starts from top.